cohortsRoutes/cohortsRouter.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express');
+const db = require('../data/dbConfig');
+
+const router = express.Router();
+
+// add cohort
+router.post('/', (req, res) => {
+  const { name } = req.body;
+
+  if(name.length) {
+    db('cohorts')
+      .insert(req.body)
+      .then( ids => {
+        db('cohorts')
```

Comment:
Pretty sure nested db calls are the 'twist' in this weeks sprint, it's really good that you are comfortable using them.